### PR TITLE
New definitions for office-js library

### PR DESCRIPTION
**Pull request by robmhoward**

### CODE REVIEW

**Comment by vvakame** [comment]
*office-js/office-js.d.ts*

check list

* [ ] is collect [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)?
  * https://www.npmjs.com/package/office-js
  * http://bower.io/search/?q=office-js
  * others?
* [X] has a [test file](http://definitelytyped.org/guides/contributing.html#tests)? (office-js/office-js-tests.ts or others)
* [ ] pass the Travis-CI test?

File: office-js/office-js-tests.ts [hunk 1]
```diff
@@ -0,0 +1,175 @@
+/*
+------------------------------------------ START OF LICENSE -----------------------------------------
```
Comment (vvakame) [inline office-js/office-js-tests.ts]: https://github.com/borisyankov/DefinitelyTyped/blob/master/LICENSE 
this repository is a MIT lisence

**Comment by robmhoward** [comment]
Removed the license from the file.  Regarding naming, the library is not on npm or bower - only available via CDN.

File: office-js/office-js-tests.ts (hunk 1 above)
Comment (vvakame) [inline office-js/office-js-tests.ts]: still have a license.

**Comment by vvakame** [comment]
https://www.npmjs.com/package/office-js is already exists.
please rename something else ... like ... `microsoft-office-js`? it is not exists on npm and bower.

**Comment by robmhoward** [comment]
I don't think renaming is going to help much. Because there are no packages for it on bower or npm, someone can come along after this is added here and use the same name in npm or bower, regardless of what we choose. I'll see what we can come up with, though.

**Comment by robmhoward** [comment]
Talked with the owner of office-js and he unpublished his project.  I think we should be good to go with the naming now.

**Comment by vvakame** [comment]
wow, okay. it merged.

**Pull request merged**